packages/frontend/src/view/pages/user/common.ts

```diff
@@ -5,25 +5,29 @@ export const getAssetsTableProps = (starkKey: StarkKey) => ({
   entryShortNamePlural: 'assets',
   entryLongNamePlural: 'assets',
   link: `/users/${starkKey.toString()}/assets`,
+  description: `Shows table of assets for ${starkKey.toString()} user`,
 })
 
 export const getBalanceChangeTableProps = (starkKey: StarkKey) => ({
   title: 'Balance changes',
   entryShortNamePlural: 'changes',
   entryLongNamePlural: 'balance changes',
   link: `/users/${starkKey.toString()}/balance-changes`,
+  description: `Shows table of balance changes for ${starkKey.toString()} user`,
 })
 
 export const getTransactionTableProps = (starkKey: StarkKey) => ({
   title: 'Ethereum transactions',
   entryShortNamePlural: 'transactions',
   entryLongNamePlural: 'ethereum transactions',
   link: `/users/${starkKey.toString()}/transactions`,
+  description: `Shows table of ethereum transactions for ${starkKey.toString()} user`,
```

Comment:
Don't use the "Shows...." text in descriptions. The `description` meta in the page header usually just says what is displayed. So in this case description should be e.g.:

"Ethereum transactions of user ${starkKey.toString()}"

Please update all other descriptions accordingly.